--- android-ndk-r26d/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/grp.h	2026-04-17 05:41:10.347353894 +0000
+++ android-ndk-r26d/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/grp.h	2026-04-17 05:41:53.464354843 +0000
@@ -56,7 +56,7 @@
 struct group* _Nullable getgrent(void) __INTRODUCED_IN(26);
 
 void setgrent(void) __INTRODUCED_IN(26);
-void endgrent(void) __INTRODUCED_IN(26);
+// void endgrent(void) __INTRODUCED_IN(26);
 #endif /* __ANDROID_API__ >= 26 */
 
 
@@ -68,6 +68,8 @@
 int getgrouplist(const char* _Nonnull __user, gid_t __group, gid_t* __BIONIC_COMPLICATED_NULLNESS __groups, int* _Nonnull __group_count);
 int initgroups(const char* _Nonnull __user, gid_t __group);
 
+static void endgrent(void) {}
+
 __END_DECLS
 
 #endif
